Python/activation.py

```diff
@@ -3,7 +3,18 @@
 
 e = math.e
 
-class ReLU:
+class Activation:

```

Comment:
it would be better if the  'Activation' class could be marked as abstract class using the ABC module.
```suggestion
class Activation:

```